# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# 
# Translators:
# Tyler Durden <virii@enn.lu>, 2016
msgid ""
msgstr ""
"Project-Id-Version: Tor Project\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2019-01-02 09:26+0100\n"
"PO-Revision-Date: 2019-01-02 13:32+0000\n"
"Last-Translator: carolyn <carolyn@anhalt.org>\n"
"Language-Team: Luxembourgish (http://www.transifex.com/otf/torproject/language/lb/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: lb\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: ../tails_installer/creator.py:100
msgid "You must run this application as root"
msgstr "Dës Applikatioun muss als root ausgefouert ginn"

#: ../tails_installer/creator.py:146
msgid "Extracting live image to the target device..."
msgstr "Live Image op Zil Medium extrahéieren..."

#: ../tails_installer/creator.py:153
#, python-format
msgid "Wrote to device at %(speed)d MB/sec"
msgstr "Mat %(speed)d MB/sek op d'Medium geschriwwen"

#: ../tails_installer/creator.py:296
#, python-format
msgid ""
"There was a problem executing the following command: `%(command)s`.\n"
"A more detailed error log has been written to '%(filename)s'."
msgstr "Et gouf e Problem beim ausféiere vu folgendem Kommando: `%(command)s`.\nEng méi detailléiert Feelerlogdatei gouf an '%(filename)s' geschriwwen."

#: ../tails_installer/creator.py:315
msgid "Verifying SHA1 checksum of LiveCD image..."
msgstr "SHA1 Checkzomm vun der LiveCD verifizéieren..."

#: ../tails_installer/creator.py:319
msgid "Verifying SHA256 checksum of LiveCD image..."
msgstr "SHA256 Checkzomm vun der LiveCD verifizéieren..."

#: ../tails_installer/creator.py:335
msgid ""
"Error: The SHA1 of your Live CD is invalid.  You can run this program with "
"the --noverify argument to bypass this verification check."
msgstr "Feeler: SHA1 Checkzomm vun der Live CD ass falsch. Dir kennt dëst Programm mat dem --noverify Argument starten em de Verifikatiounscheck ze ëmgoen."

#: ../tails_installer/creator.py:341
msgid "Unknown ISO, skipping checksum verification"
msgstr "Onbekannten ISO, Checkzomm Verifikatioun gëtt iwwersprongen"

#: ../tails_installer/creator.py:353
#, python-format
msgid ""
"Not enough free space on device.\n"
"%dMB ISO + %dMB overlay > %dMB free space"
msgstr "Net genuch fräi Späicherplaz op dem Medium.\n%dMB ISO + %dMB Späicherplaz benéidegt > %dMB fräie Späicher"

#: ../tails_installer/creator.py:360
#, python-format
msgid "Creating %sMB persistent overlay"
msgstr "%sMB u persistenter Späicherplaz goufen erstallt."

#: ../tails_installer/creator.py:421
#, python-format
msgid "Unable to copy %(infile)s to %(outfile)s: %(message)s"
msgstr "Kopéiere vun %(infile)s no %(outfile)s ass schif gaangen: %(message)s"

#: ../tails_installer/creator.py:435
msgid "Removing existing Live OS"
msgstr "Läschen existéierend Live OS"

#: ../tails_installer/creator.py:444 ../tails_installer/creator.py:457
#, python-format
msgid "Unable to chmod %(file)s: %(message)s"
msgstr "chmod vun %(file)s net méiglech: %(message)s"

#: ../tails_installer/creator.py:450
#, python-format
msgid "Unable to remove file from previous LiveOS: %(message)s"
msgstr "Datei vum leschte LiveOS konnt net geläscht ginn: %(message)s"

#: ../tails_installer/creator.py:464
#, python-format
msgid "Unable to remove directory from previous LiveOS: %(message)s"
msgstr "Verzeechnes vum leschte LiveOS konnt net geläscht ginn: %(message)s"

#: ../tails_installer/creator.py:512
#, python-format
msgid "Cannot find device %s"
msgstr "Kann d'Maschinn %s net fannen"

#: ../tails_installer/creator.py:713
#, python-format
msgid "Unable to write on %(device)s, skipping."
msgstr "Net méiglech op %(device)s ze schreiwen, iwwersprangen."

#: ../tails_installer/creator.py:743
#, python-format
msgid ""
"Some partitions of the target device %(device)s are mounted. They will be "
"unmounted before starting the installation process."
msgstr "E puer Partitioune vum ausgewielte Medium %(device)s si gemount. Se wäerten ausgehaange ginn ier den Installatiounsprozess ufänkt."

#: ../tails_installer/creator.py:786 ../tails_installer/creator.py:1010
msgid "Unknown filesystem.  Your device may need to be reformatted."
msgstr "Onbekanntent Dateisystem. D'Medium muss néi formatéiert ginn."

#: ../tails_installer/creator.py:789 ../tails_installer/creator.py:1013
#, python-format
msgid "Unsupported filesystem: %s"
msgstr "Dateisystem net ënnerstëtzt: %s"

#: ../tails_installer/creator.py:807
#, python-format
msgid "Unknown GLib exception while trying to mount device: %(message)s"
msgstr ""

#: ../tails_installer/creator.py:812
#, python-format
msgid "Unable to mount device: %(message)s"
msgstr "Medium konnt net agehaangen ginn: %(message)s"

#: ../tails_installer/creator.py:817
msgid "No mount points found"
msgstr "Keng Plaz zum Mounten fonnt"

#: ../tails_installer/creator.py:828
#, python-format
msgid "Entering unmount_device for '%(device)s'"
msgstr "unmount_devide fir '%(device)s'"

#: ../tails_installer/creator.py:838
#, python-format
msgid "Unmounting mounted filesystems on '%(device)s'"
msgstr "Aushänke vum agehaangenem Dateisystem op '%(device)s'"

#: ../tails_installer/creator.py:842
#, python-format
msgid "Unmounting '%(udi)s' on '%(device)s'"
msgstr "Aushänke vu '%(udi)s' op '%(device)s'"

#: ../tails_installer/creator.py:853
#, python-format
msgid "Mount %s exists after unmounting"
msgstr "Mount %s existéiert nach no dem aushänken."

#: ../tails_installer/creator.py:866
#, python-format
msgid "Partitioning device %(device)s"
msgstr "Späichermedium %(device)s gëtt partitionéiert"

#: ../tails_installer/creator.py:995
#, python-format
msgid "Unsupported device '%(device)s', please report a bug."
msgstr "Medium net ënnerstëtzt '%(device)s'. Mellt wann ech gelift de Feeler."

#: ../tails_installer/creator.py:998
msgid "Trying to continue anyway."
msgstr "Versichen trotzdem weider ze maachen."

#: ../tails_installer/creator.py:1007 ../tails_installer/creator.py:1405
msgid "Verifying filesystem..."
msgstr "Dateisystem verifizéieren..."

#: ../tails_installer/creator.py:1031
#, python-format
msgid "Unable to change volume label: %(message)s"
msgstr "Ännerung vun der Volumebeschreiwung net méiglech: %(message)s"

#: ../tails_installer/creator.py:1037 ../tails_installer/creator.py:1440
msgid "Installing bootloader..."
msgstr "Bootloader gëtt installéiert..."

#: ../tails_installer/creator.py:1064
#, python-format
msgid "Could not find the '%s' COM32 module"
msgstr ""

#: ../tails_installer/creator.py:1072 ../tails_installer/creator.py:1458
#, python-format
msgid "Removing %(file)s"
msgstr "Läschen %(file)s"

#: ../tails_installer/creator.py:1186
#, python-format
msgid "%s already bootable"
msgstr "%s ass schonns bootbar"

#: ../tails_installer/creator.py:1206
msgid "Unable to find partition"
msgstr "Et gouf keng Partitioun fonnt"

#: ../tails_installer/creator.py:1229
#, python-format
msgid "Formatting %(device)s as FAT32"
msgstr "%(device)s gëtt als FAT32 formatéiert."

#: ../tails_installer/creator.py:1289
msgid "Could not find syslinux' gptmbr.bin"
msgstr ""

#: ../tails_installer/creator.py:1302
#, python-format
msgid "Reading extracted MBR from %s"
msgstr ""

#: ../tails_installer/creator.py:1306
#, python-format
msgid "Could not read the extracted MBR from %(path)s"
msgstr ""

#: ../tails_installer/creator.py:1319 ../tails_installer/creator.py:1320
#, python-format
msgid "Resetting Master Boot Record of %s"
msgstr "Master Boot Record vun %s gëtt zeréck gesat"

#: ../tails_installer/creator.py:1325
msgid "Drive is a loopback, skipping MBR reset"
msgstr "D'Lafwierk ass e Loopback, MBR zerécksetze gëtt iwwersprongen"

#: ../tails_installer/creator.py:1329 ../tails_installer/creator.py:1589
#, python-format
msgid "Calculating the SHA1 of %s"
msgstr "SHA1 Zomm vun %s gëtt berechent"

#: ../tails_installer/creator.py:1354
msgid "Synchronizing data on disk..."
msgstr "Daten op dem Disk gi synchroniséiert..."

#: ../tails_installer/creator.py:1397
msgid "Error probing device"
msgstr "Feeler beim iwwerpréiwe vum Späichermedium"

#: ../tails_installer/creator.py:1399
msgid "Unable to find any supported device"
msgstr "Et konnt kee Medium fonnt gi wat ënnerstëtzt gëtt"

#: ../tails_installer/creator.py:1409
msgid ""
"Make sure your USB key is plugged in and formatted with the FAT filesystem"
msgstr "Stellt sécher, datt den USB Stick agestach ass an FAT formatéiert ass"

#: ../tails_installer/creator.py:1412
#, python-format
msgid ""
"Unsupported filesystem: %s\n"
"Please backup and format your USB key with the FAT filesystem."
msgstr "Dateisystem net ënnerstëtzt: %s\nMacht ee Backup a formatéiert den USB Stick mat dem FAT Dateisystem."

#: ../tails_installer/creator.py:1481
msgid ""
"Unable to get Win32_LogicalDisk; win32com query did not return any results"
msgstr "Win32_LogicalDisk konnt net ermëttelt ginn; de win32com Opruff huet keng Resultater bruecht."

#: ../tails_installer/creator.py:1536
msgid "Cannot find"
msgstr "Kann net fonnt ginn"

#: ../tails_installer/creator.py:1537
msgid ""
"Make sure to extract the entire tails-installer zip file before running this"
" program."
msgstr ""

#: ../tails_installer/gui.py:69
#, python-format
msgid "Unknown release: %s"
msgstr "Onbekannt release: %s"

#: ../tails_installer/gui.py:73
#, python-format
msgid "Downloading %s..."
msgstr "%s gëtt erofgelued"

#: ../tails_installer/gui.py:213
msgid ""
"Error: Cannot set the label or obtain the UUID of your device.  Unable to "
"continue."
msgstr "Feeler: Kann d'Bezeechnung oder UUID vum Medium net ermëttelen. Weiderfueren ass net méiglech."

#: ../tails_installer/gui.py:260
#, python-format
msgid "Installation complete! (%s)"
msgstr "Installatioun ofgeschloss! (%s)"

#: ../tails_installer/gui.py:265
msgid "Tails installation failed!"
msgstr ""

#: ../tails_installer/gui.py:369
msgid ""
"Warning: This tool needs to be run as an Administrator. To do this, right "
"click on the icon and open the Properties. Under the Compatibility tab, "
"check the \"Run this program as an administrator\" box."
msgstr "Warnung: Dëst Tool muss als Administrateur ausgefouert ginn. Fir dat ze maachen, musst der op der Maus op den Icon riets klicken an dann d'Astellungen auswielen. Ënnert dem Reider \"Kompatibilitéit\", wielt der dann \"Dëst Programm als Administrateur ausféieren\" aus."

#: ../tails_installer/gui.py:381
msgid "Tails Installer"
msgstr ""

#: ../tails_installer/gui.py:441
msgid "Tails Installer is deprecated"
msgstr ""

#: ../tails_installer/gui.py:443
msgid ""
"To install a new Tails, see the <a href='https://tails.boum.org/install/'>installation documentation</a>.\n"
"To upgrade an existing Tails, see the <a href='https://tails.boum.org/upgrade/'>upgrade documentation</a>."
msgstr ""

#: ../tails_installer/gui.py:450 ../data/tails-installer.ui.h:2
msgid "Clone the current Tails"
msgstr ""

#: ../tails_installer/gui.py:457 ../data/tails-installer.ui.h:3
msgid "Use a downloaded Tails ISO image"
msgstr ""

#: ../tails_installer/gui.py:494 ../tails_installer/gui.py:815
msgid "Upgrade"
msgstr ""

#: ../tails_installer/gui.py:496
msgid "Manual Upgrade Instructions"
msgstr ""

#: ../tails_installer/gui.py:498
msgid "https://tails.boum.org/upgrade/"
msgstr ""

#: ../tails_installer/gui.py:506 ../tails_installer/gui.py:727
#: ../tails_installer/gui.py:792 ../data/tails-installer.ui.h:7
msgid "Install"
msgstr ""

#: ../tails_installer/gui.py:509 ../data/tails-installer.ui.h:1
msgid "Installation Instructions"
msgstr ""

#: ../tails_installer/gui.py:511
msgid "https://tails.boum.org/install/"
msgstr ""

#: ../tails_installer/gui.py:517
#, python-format
msgid "%(size)s %(vendor)s %(model)s device (%(device)s)"
msgstr ""

#: ../tails_installer/gui.py:529
msgid "No ISO image selected"
msgstr ""

#: ../tails_installer/gui.py:530
msgid "Please select a Tails ISO image."
msgstr ""

#: ../tails_installer/gui.py:572
msgid "No device suitable to install Tails could be found"
msgstr ""

#: ../tails_installer/gui.py:574
#, python-format
msgid "Please plug a USB flash drive or SD card of at least %0.1f GB."
msgstr ""

#: ../tails_installer/gui.py:608
#, python-format
msgid ""
"The USB stick \"%(pretty_name)s\" is configured as non-removable by its "
"manufacturer and Tails will fail to start on it. Please try installing on a "
"different model."
msgstr ""

#: ../tails_installer/gui.py:618
#, python-format
msgid ""
"The device \"%(pretty_name)s\" is too small to install Tails (at least "
"%(size)s GB is required)."
msgstr "D'Medium \"%(pretty_name)s\" ass ze kleng fir Tails dorobber ze installéieren (et ginn op mannst %(size)s GB gebraucht)."

#: ../tails_installer/gui.py:631
#, python-format
msgid ""
"To upgrade device \"%(pretty_name)s\" from this Tails, you need to use a downloaded Tails ISO image:\n"
"https://tails.boum.org/install/download"
msgstr ""

#: ../tails_installer/gui.py:652
msgid "An error happened while installing Tails"
msgstr ""

#: ../tails_installer/gui.py:664
msgid "Refreshing releases..."
msgstr "Verëffentlechunge ginn aktualiséiert."

#: ../tails_installer/gui.py:669
msgid "Releases updated!"
msgstr "Verëffentlechunge goufen aktualiséiert!"

#: ../tails_installer/gui.py:722
msgid "Installation complete!"
msgstr "Installatioun ofgeschloss!"

#: ../tails_installer/gui.py:738
msgid "Cancel"
msgstr "Ofbriechen"

#: ../tails_installer/gui.py:774
msgid "Unable to mount device"
msgstr "Medium konnt net agehaange ginn"

#: ../tails_installer/gui.py:781 ../tails_installer/gui.py:814
msgid "Confirm the target USB stick"
msgstr ""

#: ../tails_installer/gui.py:782
#, python-format
msgid ""
"%(size)s %(vendor)s %(model)s device (%(device)s)\n"
"\n"
"All data on this USB stick will be lost."
msgstr ""

#: ../tails_installer/gui.py:801
#, python-format
msgid "%(parent_size)s %(vendor)s %(model)s device (%(device)s)"
msgstr ""

#: ../tails_installer/gui.py:809
msgid ""
"\n"
"\n"
"The persistent storage on this USB stick will be preserved."
msgstr ""

#: ../tails_installer/gui.py:810
#, python-format
msgid "%(description)s%(persistence_message)s"
msgstr ""

#: ../tails_installer/gui.py:853
msgid "Download complete!"
msgstr "Download fäerdeg!"

#: ../tails_installer/gui.py:857
msgid "Download failed: "
msgstr "Download feel geschloen"

#: ../tails_installer/gui.py:858
msgid "You can try again to resume your download"
msgstr "Download probéieren ze resuméieren"

#: ../tails_installer/gui.py:866
msgid ""
"The selected file is unreadable. Please fix its permissions or select "
"another file."
msgstr "Déi ausgewielten Datei ass net liesbar. Ännert wann ech gelift är Berechtegungen oder wielt eng aner Datei aus."

#: ../tails_installer/gui.py:872
msgid ""
"Unable to use the selected file.  You may have better luck if you move your "
"ISO to the root of your drive (ie: C:\\)"
msgstr "Net méiglech déi ausgewielten Datei ze benotzen. Dir hutt wuel méi Gléck wann dir är ISO direkt an engem Lafwierk ofleet (zB: C:\\)"

#: ../tails_installer/gui.py:878
#, python-format
msgid "%(filename)s selected"
msgstr "%(filename)s ausgewielt"

#: ../tails_installer/source.py:28
msgid "Unable to find LiveOS on ISO"
msgstr ""

#: ../tails_installer/source.py:34
#, python-format
msgid "Could not guess underlying block device: %s"
msgstr ""

#: ../tails_installer/source.py:49
#, python-format
msgid ""
"There was a problem executing `%s`.\n"
"%s\n"
"%s"
msgstr ""

#: ../tails_installer/source.py:63
#, python-format
msgid "'%s' does not exist"
msgstr ""

#: ../tails_installer/source.py:65
#, python-format
msgid "'%s' is not a directory"
msgstr ""

#: ../tails_installer/source.py:75
#, python-format
msgid "Skipping '%(filename)s'"
msgstr ""

#: ../tails_installer/utils.py:44
#, python-format
msgid ""
"There was a problem executing `%s`.%s\n"
"%s"
msgstr ""

#: ../tails_installer/utils.py:124
msgid "Could not open device for writing."
msgstr ""

#: ../data/tails-installer.ui.h:4
msgid "Select a distribution to download:"
msgstr ""

#: ../data/tails-installer.ui.h:5
msgid "Target USB stick:"
msgstr ""

#: ../data/tails-installer.ui.h:6
msgid "Reinstall (delete all data)"
msgstr ""
